lighthouse-core/report/v2/report-template.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+  <title>Lighthouse Report</title>
+</head>
+<body>
+  <noscript>Lighthouse report requires JavaScript. Please enable.</noscript>
+  <script>window.__LIGHTHOUSE_JSON__ = %%LIGHTHOUSE_JSON%%</script>
+  <script>%%LIGHTHOUSE_JAVASCRIPT%%</script>
```

Comment:
devtools probably won't use this at all actually, it should just include the renderer source files directly, this is pretty much exclusively for saving the report to an html file